### PR TITLE
Slight changes

### PR DESCRIPTION
**Pull request by saundefined**
Corrected a typo, added several Russian messages, the year in copyright is not hardcoded

### CODE REVIEW

File: js/ui/cmp/About.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ ui.cmp.About = Ext.extend(Ext.Window,
                             '</div>' +
                             '<div id="phd-oe-about-info">' + PhDOE.app.name + //' ver ' + PhDOE.app.ver + '<br/>' +
                                 //'UI: ' + PhDOE.app.uiRevision + '<br/>' +
-                                ' Copyright &copy; 2008-2012 The PHP Group<br/>' +
+                                ' Copyright &copy; 2008-' + new Date().format('Y') + ' The PHP Group<br/>' +
```
Comment (salathe) [inline js/ui/cmp/About.js]: Suggest [`new Date().getFullyear()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getFullYear).

Comment (saundefined) [reply js/ui/cmp/About.js]: @salathe thanks, fixed it.
The previous format was similar to [this code](https://github.com/php/web-doc-editor/blob/master/js/ui/cmp/PortletInfo.js#L288)

Comment (salathe) [reply js/ui/cmp/About.js]: Ah, I forgot that ExtJS adds that method.  Feel free to use whichever you choose. 😄

Comment (saundefined) [reply js/ui/cmp/About.js]: Hmm ... for a uniform code style, the format will probably be better?
If you agree with me - I'll send a new commit :)

Comment (salathe) [reply js/ui/cmp/About.js]: Sure, that sounds fine.

Comment (saundefined) [reply js/ui/cmp/About.js]: done :)

**Comment by php-pulls** [comment]
**Comment on behalf of salathe at php.net:**

Merged as php/web-doc-editor@58d9916776

**Pull request closed without merging**